_layouts/default.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="{{ site.lang | default: "en-US" }}">
+  <head>
+
+    <meta charset="UTF-8">
+
+{% seo %}
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#157878">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+  </head>
+  <body>
+
+    <header class="page-header" role="banner">
+      <h1 class="project-name">{{ page.title | default: site.title | default: site.github.repository_name }}</h1>
+      <h2 class="project-tagline">{{ page.description | default: site.description | default: site.github.project_tagline }}</h2>
+      {% if site.github.is_project_page %}
+        <a href="https://github.com/indico/newdle/" class="btn btn-default btn-lg">View on GitHub</a>
+        <a href="https://github.com/indico/newdle/#development" class="btn btn-default btn-lg">Install instructions</a>
+      {% endif %}
+     </header>
+
+      <main id="content" class="main-content" role="main">
+          {{ content }}
+
+       <footer class="site-footer">
+        {% if site.github.is_project_page %}
+          <span class="site-footer-owner"><a href="https://newdle.cern.ch/">Newdle</a> is maintained by the <a href="https://indico.cern.ch/">Indico Team</a>.</span>
```

Comment:
I don't think linking to the CERN instances is a good idea here - especially newdle is not useful for someone without a CERN account, and indico is not directly relevant for someone interested in the newdle project. Maybe link to the newdle repo and indico org instead?

Reply:
What do you mean with `indico org`?

Reply:
https://github.com/indico/